vroom/main.vroom

```diff
@@ -1,7 +1,16 @@
 This file demonstrates the basics of coverage usage.
 
 In order for these tests to work, maktaba MUST be in the same parent directory
-as coverage. Given that that's the case, all we have to do is source the
+as coverage.
+
+First, we need to work around for nvim prompting users to press enter.
```

Comment:
This could go in the setupvroom.vim file so it doesn't distract from these vroom contents.